pkg/controller/storageclusterinitialization/noobaa_system_reconciler.go

```diff
@@ -57,8 +59,12 @@ func (r *ReconcileStorageClusterInitialization) newNooBaaSystem(initialData *ocs
 		},
 
 		Spec: nbv1.NooBaaSpec{
-			StorageClassName: &storageClassName,
-			CoreResources:    &coreResources,
+			DBStorageClass:            &storageClassName,
+			PVPoolDefaultStorageClass: &storageClassName,
+			CoreResources:             &coreResources,
+			DBResources:               &dbResources,
+			Tolerations:               defaults.DaemonPlacements["noobaa-core"].Tolerations,
+			DBVolumeResources:         &dBVolumeResources,
```

Comment:
what would happen if this dbVolumeResources is empty?

Reply:
It will make the PVC resources empty which probably means it will fail?
Is there a case when it is empty?

Reply:
no, not unless someone explicitly sets the noobaa-db-vol request requirements to empty lists.

I had originally thought that was happening in our CI environment, but now I realize we're explicitly overriding entries by name, so we'd have to expliclity set the noobaa-db-vol to empty lists for this to be an issue... which we aren't